src/main/java/com/contrastsecurity/sdk/ContrastSDK.java

```diff
@@ -151,7 +160,7 @@ public ContrastSDK(String user, String serviceKey, String apiKey, String restApi
         ContrastSDKUtils.validateUrl(this.restApiURL);
         this.restApiURL = ContrastSDKUtils.ensureApi(this.restApiURL);
         this.urlBuilder = UrlBuilder.getInstance();
-        this.gson = new GsonBuilder()
+        this.gson = new GsonBuilder().excludeFieldsWithoutExposeAnnotation()

```

Comment:
I think this breaks serialized objects that were never tagged with expose like JobOutcomePolicyListResponse

Reply:
Maybe we should us transient instead of expose annotation? what do you think?